crates/consensus/src/transaction/request.rs

```diff
@@ -0,0 +1,256 @@
+use alloy_network::Transaction;
+use alloy_primitives::Signature;
+use alloy_rlp::Encodable;
+
+use crate::{TxEip1559, TxEip2930, TxEnvelope, TxLegacy, TxType};
+
+/// The TypedTransactionRequest enum represents all Ethereum transaction request types.
+///
+/// Its variants correspond to specific allowed transactions:
+/// 1. Legacy (pre-EIP2718) [`TxLegacy`]
+/// 2. EIP2930 (state access lists) [`TxEip2930`]
+/// 3. EIP1559 [`TxEip1559`]
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub enum TypedTransactionRequest {
```

Comment:
this should not be called request, just tx

Reply:
done in https://github.com/alloy-rs/alloy/pull/183/commits/27429615200dc2fde2a2c88d97c332178544bde0